core/src/scss/components/main-nav/_main-nav--dark.scss

```diff
@@ -1,125 +1,127 @@
 @charset "UTF-8";
 
-.su-main-nav--dark {
-  @include grid-media-max('md') {
-    .su-main-nav__toggle {
-      color: $color-white;
-
-      &:hover,
-      &:focus {
-        &::before {
-          background-color: $color-white;
-        }
-      }
-
-      &:active {
-        &::before {
-          background-color: $color-vivid-red;
+.su-main-nav {
```

Comment:
It's too bad you had to do that. I would prefer to see that we don't double up on the modifier classes when possible.   Using the CSS cascade to have modifiers come after the main styles can be enough to have more specificity. I would challenge you to try to take another look and see if you can refactor this to win out over the main styles without the double class selector.

Reply:
Yes, I agree with you! Sorry I didn't see this comment before I merged. I would definitely go back and fine tune it when I have a chance.

Reply:
You are fast on the trigger 🔫! Yeah, this comment doesn't block the merge but it is something that could use attention down the line and not a pattern I want to see too often.